### ISSUE TITLE
Toast doesn't announce on NVDA

### ISSUE THREAD

**Issue by argyleink** [argyleink/gui-challenges#78]
https://twitter.com/KonstantinRouda/status/1473668440670060545?t=7lKwyJASF8ZuOf2bFPu8rw&s=19
